doc/lsst.pipe.base/command-line-task-argument-reference.rst

```diff
@@ -0,0 +1,353 @@
+
+.. option:: --selectId
+
+   **Input data IDs for many-to-one tasks.**
+
+   For many-to-one processing tasks, such as coaddition, the :option:`--selectId` argument is used to specify input data IDs, while :option:`--id` is used to specify *output* data IDs.
+   The syntax for :option:`--selectId` is identical to that of :option:`--id`.
+
+   For more information about dataId selection syntax, see :ref:`command-line-task-dataid-howto`.
+
+.. option:: -t timeout, --timeout timeout
+
+   **Multiprocessing timeout (in seconds).***
+
+   See also :option:`-j`.
+
+.. option:: --noExit
+
+   **Prevent the command-line task from exiting early with a non-zero status code if there are one or more processing failures.**
+
+   The normal command-line task behavior is to exit with a status code equal to the number of data IDs that it failed to process.
+   This requires a command-line task to exit early, within `lsst.pipe.base.ParseAndRun`.
+
+   When ``--noExit`` is used, the command-line task will not exit in `lsst.pipe.base.ParseAndRun` if failures are encountered, and continue to return normally to the script that called `~lsst.pipe.base.ParseAndRun`.
+   In this case, it is up to the calling script to set an appropriate status code.
+
+.. _command-line-argument-files:
+
+Argument files
+==============
+
+Arguments can be written to a plain text file and referenced with an ``@filepath`` command-line argument.
+The contents of argument files are identical to what you'd write on the command line, with these rules:
+
+- Text can be split across multiple lines.
+  For example, you can have one argument per line.
+
+- Do not use ``\`` as a continuation character.
+
+- Include comments with a ``#`` character.
+  Content on a line after the ``#`` character is ignored.
+
+- Blank lines and lines starting with ``#`` are ignored.
+
+You can mix argument files with other command-line arguments (including additional :option:`--id` and :option:`--config` arguments).
+
+You can include multiple ``@filepath`` references in the same command.
+
+Example
+-------
+
+For example, the file :file:`foo.txt` contains:
+
+.. code-block:: text
+
+   --id visit=54123^55523 raft=1,1^2,1 # data ID
+   --config someParam=someValue --configfile configOverrideFilePath
+
+You can then reference it with ``@foo.txt``, along with additional command-line arguments:
+
+.. code-block:: bash
+
+   task.py repo @foo.txt --config anotherParam=anotherValue --output outputPath
+
+.. _command-line-task-envvar:
+
+Environment variables
+=====================
+
+The :envvar:`PIPE_INPUT_ROOT`, :envvar:`PIPE_CALIB_ROOT`, and :envvar:`PIPE_OUTPUT_ROOT` environment variables let you more easily specify Butler data repositories.
+
+Each environment variable is used as a root directory for relative paths provided on the command line.
+If you set an absolute path on the command line, the environment variable is ignored.
+:ref:`see examples <command-line-task-envvar-examples>`.
```

Comment:
This reference isn't very useful since the examples are just a few lines down the page.